src/logics/LIALogic.cc

```diff
@@ -249,3 +204,67 @@ PTRef LIALogic::sumToNormalizedInequality(PTRef sum) {
     constantValue = constantValue.ceil();
     return mkFun(get_sym_Num_LEQ(), {mkConst(constantValue), normalizedSum});
 }
+
+PTRef LIALogic::insertTerm(SymRef sym, vec<PTRef> &terms) {
+    if (sym == get_sym_Int_MOD())
+        return mkIntMod(terms);
+    if (sym == get_sym_Int_DIV())
+        return mkIntDiv(terms);
+    return LALogic::insertTerm(sym, terms);
+}
+
+PTRef LIALogic::mkIntDiv(vec<PTRef> const & args) {
+    if (args.size() != 2) { throw OsmtApiException("Incorrect number of arguments for DIV operator"); }
+    return _mkIntDiv(args);
```

Comment:
Div is defined as `(div Int Int Int :left-assoc)`.  Maybe it wouldn't be too much work to split it here into binary calls?

Reply:
We'll do this later in a separate PR.